### PR TITLE
Translated this README file to Russian

### PR DESCRIPTION
**Pull request by Boto1**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

#18308

### CODE REVIEW

**Comment by Boto1** [comment]
#18308

**Review by PurpleCatfish — APPROVED** [review]

File: docs/russian/README.md [hunk 1]
```diff
@@ -12,41 +12,39 @@
...
 | ------- | ----------- |
-| `npm run bootstrap` | Bootstraps the different services |
-| `npm run seed` | Parse all the challenge markdown files and inserts them into MongoDB. |
-| `npm run develop` | Starts the freeCodeCamp API Server and Client Apps |
-| `npm test` |  Run all JS tests in the system, including client, server, lint and challenge tests |
-| `npm run test:client` | Run the client test suite |
-| `npm run test:curriculum` | Run the curriculum test suite |
-| `npm run test:server` | Run the server test suite |
-| `npm run commit` | An interactive tool to help you build a good commit message |
+| `npm run bootstrap` | Загружает различные сервисы. |
```
Comment (sergeyradov) [inline docs/russian/README.md]: Something happened to table formatting. The table in the place looks broken in comparison to English Verison.

**Review by PurpleCatfish — CHANGES_REQUESTED** [review]
LGTM

**Comment by sergeyradov** [comment]
@Boto1 Could you please rebase on the master branch to resolve the conflict?

**Review by greggubarev — APPROVED** [review]
**Thank you for opening this pull request.**

This is a standard message notifying you that we’ve reviewed your pull request and have decided not to merge it. **We would welcome future pull requests from you.**

Thank you and happy coding.

**Pull request closed without merging**